simple-wp-helpdesk/assets/src/theme-violet/components.scss

```
@@ -0,0 +1,170 @@
+.swh-theme-violet .swh-helpdesk-wrapper button[type="submit"],
+.swh-theme-violet .swh-helpdesk-wrapper .swh-portal-cta {
+    background: var(--swh-primary);
+    color: var(--swh-primary-fg);
+    border-radius: var(--swh-radius-md);
+    box-shadow: var(--swh-shadow-sm);
+}
+
+/* Legacy components — keep visually consistent even though they predate v4.0 */
+.swh-theme-violet .swh-badge {
+    border-radius: var(--swh-radius-full);
+    font-family: 'Geist Sans', 'Geist', -apple-system, sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)